themes/bootstrap3/less/components/record.less

```diff
@@ -217,6 +217,12 @@
 .tab-pane .result {
   margin-left: 0;
 }
+.holdings-tab table {
```

Comment:
I know you suggested a class name on the table itself, but I think this does the trick?  If there are counter-examples of tables on the holdings tab that should be treated differently, then I can narrow it down.